sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -328,24 +277,24 @@ q.parallel_for(nd_range<2>(G, L), [=](nd_item<2> item)
    const auto sg_startx = global_idx - item.get_local_id(0);
    const auto sg_starty = global_idy - item.get_local_id(1);
    sub_group sg = item.get_sub_group();
-   joint_matrix<int8_t, use::a, tM, tK, layout::row_major> tA(sg);
-   joint_matrix<int8_t, use::b, tK, tN, layout::row_major> tB(sg);
-   joint_matrix<int32_t, use::accumulator, tM, tN> tC(sg);
+   joint_matrix<sub_group, int8_t, use::a, tM, tK, layout::row_major> tA();
+   joint_matrix<sub_group, int8_t, use::b, tK, tN, layout::row_major> tB();
+   joint_matrix<sub_group, int32_t, use::accumulator, tM, tN> tC();
    joint_matrix_fill(sg, tC, 0);
    for (int k = 0; k < K; k += tk) {
      joint_matrix_load(sg, tA, memA + sg_startx * tM * K + k, K);
      joint_matrix_load(sg, tB, memB + k * N + sg_starty/SG_SIZE*tN, N); 
```

Comment:
```suggestion
     joint_matrix_load(sg, tB, multi_ptr<int8_t, sycl::access::address_space::global_space>(memB + k * N + sg_starty/SG_SIZE*tN), N); 
```

Reply:
@jsji this is actually a bug. The implementation should be able to handle both types of pointers. 
Thank you for spotting this. I will report this bug and create a test for this.